manubot/pandoc/tests/test_cite_filter/input.md

```diff
@@ -11,6 +11,7 @@ references:
   container-title: Greene Laboratory
   location: University of Pennsylvania
   issued: {'date-parts': [[2018, 12, 19]]}
+# manubot-bibliography-cache: manubot/pandoc/tests/test_cite_filter/bibliography-cache.json
```

Comment:
Why is this commented out?

Reply:
For this test, don't want to use a CSL-bibliography cache. Uses --bibliography instead.